packages/flame/lib/src/extensions/rect.dart

```diff
@@ -83,8 +83,7 @@ extension RectExtension on Rect {
     return Rect.fromPoints(Offset(minX, minY), Offset(maxX, maxY));
   }
 
-  /// Constructs a rectangle from its center point (specified as a Vector2),
-  /// width and height.
+  /// Constructs a [Rect] with a [width] and [height] around the [center] point.
   static Rect fromVector2Center({
```

Comment:
I kinda want to rename this one because I think it was forgotten in the `Vector2` migration. But that would be a breaking change (unless we deprecate it and proxy it to the renamed one).

Reply:
Do you want to do that change in another PR?

Reply:
Lets do that in a follow up if the rest of the team agrees.